kafka-rest/src/test/java/io/confluent/kafkarest/response/StreamingResponseTest.java

```diff
@@ -122,6 +124,7 @@ public void testWriteToChunkedOutput() throws IOException {
             .setTopicName("topicName")
             .setPartitionId(1)
             .setOffset(1L)
+            .setErrorCode(HttpStatus.ACCEPTED_202)
```

Comment:
Why 202?

Reply:
I think this is a typo I didn't spot - was convinced I put a 200, sorry!